jaeger-core/src/main/java/io/jaegertracing/JaegerSpan.java

```diff
@@ -132,6 +136,10 @@ public JaegerSpan setBaggageItem(String key, String value) {
       return this;
     }
     synchronized (this) {
+      if (spanAlreadyFinished("set baggage item")) {
```

Comment:
strictly speaking setting baggage would work even after the span is finished

Reply:
It seems unlikely that the user would intend to do this, but do you want me to remove it?